### PR TITLE
changed endpoints

### PR DESCRIPTION
**Pull request by kssampson**
changed endpoints

### CODE REVIEW

**Review by gordon1213 — COMMENTED** [review]
Looks good if it's working properly. Remember to remove the url variable!

File: client/src/components/questions_answers/Answers.jsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const Answers = ( { questionId } ) => {
       }
     };
     const url = process.env.GIT_API_URL;
```
Comment (clee4037) [inline client/src/components/questions_answers/Answers.jsx:43]: Please remove const url as it is not needed anymore

File: client/src/containers/QuestionsAnswers.jsx [hunk 2]
```diff
@@ -18,9 +18,9 @@ const QuestionsAnswers = ( {itemId} ) => {
       }
     };
     const url = process.env.GIT_API_URL;
```
Comment (clee4037) [inline client/src/containers/QuestionsAnswers.jsx:20]: Remove

**Review by clee4037 — COMMENTED** [review]
Few comments. Other than that, looks good!

File: client/src/components/questions_answers/Answers.jsx (hunk 1 above)
Comment (kssampson) [reply client/src/components/questions_answers/Answers.jsx]: done

File: client/src/containers/QuestionsAnswers.jsx (hunk 2 above)
Comment (kssampson) [reply client/src/containers/QuestionsAnswers.jsx]: done

**Review by LukeLarson2 — APPROVED** [review]
Approved thank you for making the changes

**Pull request merged**